hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestDirectoryScanner.java

```diff
@@ -1260,4 +1260,11 @@ private void writeFile(FileSystem fs, int numFiles) throws IOException {
       DFSTestUtil.createFile(fs, filePath, 1, (short) 1, 0);
     }
   }
+
+  @Test(timeout = 3000)
+  public void testDefaultDirectoryScanThrottle() {
+    Configuration conf = new Configuration(getConfiguration());
+    assertEquals(Integer.parseInt(
+            conf.get(DFSConfigKeys.DFS_DATANODE_DIRECTORYSCAN_THROTTLE_LIMIT_MS_PER_SEC_KEY)), 1000);
```

Comment:
Please fix the checkstyle.

Reply:
thanks @tomscut for your review ,have just fixed it :)